js/copycode.js

```diff
@@ -1,4 +1,5 @@
 const codeBlocks = document.querySelectorAll("pre:has(code)");
+const headers = document.querySelectorAll('h2[id],h3[id]');

```

Comment:
This is broad enough that it is also selecting our sidebar which has an h3:

<img width="259" height="169" alt="Image" src="https://github.com/user-attachments/assets/0e8d9848-7381-4de0-8ea8-a7e3ae4094ae" />


We can scope it down to only select headers which appear inside the `<main>` tag
```suggestion
const headers = document.querySelectorAll('main :is(h2,h3)[id]');
```